### PR TITLE
Add headers file for HTTP headers

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**

### CODE REVIEW

**Review by shahsank3t — APPROVED** [review]

**Review by harshach — APPROVED** [review]

**Comment by Sachin-chaurasiya** [comment]
Do not merge this PR, need to verify if all things are working fine or not.

**Comment by Sachin-chaurasiya** [comment]
@harshach , `Content-Security-Policy` has some issue, so I will add that in separate PR.

**Pull request merged**
